tool/easy_install.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# Installs Ruby into custom /opt/rubies/ prefix.
+
+if [ $# -eq 1 ]
+then
+ build_name=$1
+ prefix=/opt/rubies/${build_name}
+ export BASERUBY=$(brew --prefix ruby)/bin/ruby
+else
+ echo "Please name the ruby build with the first program argument."
+ exit 1
+fi
+
+echo "Installing dependencies..." &&\
+ brew install autoconf openssl libyaml ruby &&\
+ echo "Done installing dependencies."
+
+echo "Starting..." &&\
+ ./autogen.sh &&\
+ mkdir -p build/ &&\
+ cd build &&\
+ ../configure\
+ --config-cache\
+ --prefix=${prefix}\
+ --with-baseruby\
+ --with-libyaml-dir=$(brew --prefix libyaml)\
+ --with-openssl-dir=$(brew --prefix openssl) &&\
+ make &&\
+ make install &&\
+ cd - &&\
```

Comment:
A shell script is run in an isolated process (unless run by `source` or `.` command) in common.
Back to the previous working directory is not needed.